src/components/BlogFeed.vue

```diff
@@ -17,21 +17,30 @@
   export default {
     name: 'BlogFeed',
     components: { BlogPost, CreationPost },
+    data: function () {
+      return {
+        newPosts: []
+      }
+    },
     props: {
       posts: { 
         type: Array, 
         default: () => [] 
         },
-
     },
     computed: {
       activePosts() { 
         return this.posts.filter((post) => post.active) 
       },
+      allPosts() {
+        return this.allPosts.filter((post) => post.active)
+      }
     },
     methods: {
       addPost (newPost) {
+        newPost.id = this.posts.length
         this.posts.push(newPost)
```

Comment:
Good thinking on with the ID!

Now we need to add it to our new posts data prop
`this.newPosts.push(newPost)`